core-libs/setup/ssr/logger/loggers/express-server-logger.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Request } from 'express';
+import { LogContext, ServerLogger } from './server-logger';
+
+export class ExpressServerLogger extends ServerLogger {
+  log(message: string, context: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.log(this.createLogMessage(message, context));
+  }
+  warn(message: string, context: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.warn(this.createLogMessage(message, context));
+  }
+  error(message: string, context: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.error(this.createLogMessage(message, context));
+  }
+
+  protected createLogMessage(message: string, context: LogContext): string {
+    const timestamp = new Date().toISOString();
+    const object = {
+      message,
+      context: {
+        timestamp,
+        ...context,
+      },
+    };
+    if (context.request) {
+      Object.assign(object.context, {
+        request: this.mapRequest(context.request),
+      });
+    }
+
+    return JSON.stringify(object);
+  }
+
+  protected mapRequest(request: Request): Record<string, any> {
+    return {
+      url: request.originalUrl,
+      render: request.res?.locals.cx.render,
+      headers: request.headers,
```

Comment:
TODO reminder: let's remove `headers` before merging it

Reply:
And replace with info from `traceparent` 🔥 😎

Reply:
Leaving link just for reference, to split `traceparent`: https://www.w3.org/TR/trace-context/#traceparent-header

Reply:
This feature won't be a part of this PR and will be implemented as a separate task: https://jira.tools.sap/browse/CXSPA-2067